src/views/v2/rsvp/ari/form/title.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * This template renders the RSVP AR form title.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/title.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<span class="tribe-tickets__rsvp-ar-form-title tribe-common-h5">
+	<?php esc_html_e( 'Main Guest', 'event-tickets' ); ?>
```

Comment:
I get a strong feeling this might be something people would want to customize further, can we add a filter for it above?